nearcore/res/example-config-gc.json

```diff
@@ -24,7 +24,8 @@
     },
     "telemetry": {
```

Comment:
I haven't seen this config before. Can you tell me what is it and how is it used? Is it identical to the default config generated by neard or to the config we upload to S3?

Reply:
I'm not sure myself how or when it is used. The config uploaded to S3 is another thing, I asked Andrei to change it manually.